packages/modules/order/src/models/order-item.ts

```diff
@@ -59,15 +66,30 @@ export default class OrderItem {
   @OrderIdIndex.MikroORMIndex()
   order_id: string
 
-  @Property({ columnType: "integer" })
-  @OrderVersionIndex.MikroORMIndex()
-  version: number
-
   @ManyToOne(() => Order, {
     persist: false,
   })
   order: Order
 
+  @ManyToOne({
+    entity: () => Return,
+    mapToPk: true,
+    fieldName: "return_id",
+    columnType: "text",
+    nullable: true,
+  })
+  @ReturnIdIndex.MikroORMIndex()
+  return_id: string | null = null
```

Comment:
question: In the migration, we add references to the other types of edits, e.g. `exchange_id` and `claim_id`. Should they not be added here too?

Reply:
Think this might apply to the other model changes as well.

Reply:
Initially I was planning to add them while developing each feature. But I already added them everywhere haha.
I'll add here too.